superset/assets/javascripts/SqlLab/components/SqlEditorLeftBar.jsx

```diff
@@ -124,6 +125,8 @@ ${this.props.queryEditor.schema}/${input}`;
   }
   render() {
     const shouldShowReset = window.location.search === '?reset=1';
+    const options = this.state.tableOptions;
+    const filterOptions = createFilterOptions({ options });
```

Comment:
For the record this line was problematic. It's been addressed here:
https://github.com/airbnb/superset/pull/2547